src/main/java/com/coinbase/prime/model/addressbook/AddressBookActivityType.java

```diff
@@ -17,6 +17,7 @@
 package com.coinbase.prime.model.addressbook;
 
 public enum AddressBookActivityType {
+    ACTIVITY_TYPE_UNKNOWN,
```

Comment:
Are these address book activities or onchain wallet activities?

Reply:
These are just Custody Activities, but are currently only used in any of the AddressBook related endpoints.

Reply:
There are two different activity types:

You can see that kind in the response here: https://docs.cdp.coinbase.com/prime/reference/primerestapi_getportfolioactivity

The other kind is the one from Custody found in this AddressBook response: https://docs.cdp.coinbase.com/prime/reference/primerestapi_createportfolioaddressbookentry

Reply:
Updated to rename this "CustodyActivityType" and moved all enums to a collective `enums` package